index.html

```diff
@@ -97,146 +97,146 @@ <h1 style= "font-family: 'Kaushan Script','Helvetica Neue',Helvetica,Arial, curs
+                                </div>
+                            </div>
+                        </a>
+                    </div>
+
+                    <div class="col-md-4 col-sm-6 events-item">
+                        <a href="http://tinyurl.com/ybsr45dn" target="blank" class="events-link">
+                            <div class="events-caption">
+                                <i class="fa fa-calendar fa-3x"></i>
+                                <h6>Workshop [1]</h6>
+                                <p class="text-muted">9 de Novembro de 2017, 19h30</p>
+                                <p class="text-muted"> CoNéctar Coworking </p>
+                                <div class="event-info">
+                                    <p>O que são as PyLadies? De que mundo elas vieram? Python, que ser estranho é esse? Venha descobrir tudo isso e muito mais nesse nosso primeiro workshop preparado com muito carinho para você!</p>
+                                </div>
+                            </div>
+                        </a>
+                    </div>
+
+                    <div class="col-md-4 col-sm-6 events-item">
+                        <a href="http://tinyurl.com/ya74g5hk" target="blank" class="events-link">
+                            <div class="events-caption">
+                                <i class="fa fa-calendar fa-3x"></i>
+                                <h6>Programação não é só coisa de meninos!</h6>
+                                <p class="text-muted">21 de Setembro de 2016</p>
+                                <p class="text-muted"> Universidade de São Paulo - Ribeirão Preto</p>
+                                <div class="event-info">
+                                    <p>Você sabia que a primeira pessoa a escrever um código foi uma mulher? Venha participar e descobrir o quão importante mulheres são no mundo da computação!</p>
+                                </div>
+                            </div>
+                        </a>
+                    </div>
 
-          </div>
-		<div class="div-btn-event">
-			<button type="button" class="btn btn-primary">
+                </div>
+                <div class="div-btn-event">
+                    <button type="button" class="btn btn-primary">
               			<a href="eventos.html" target="_blank">Ver mais Eventos</a>
             		</button>
-		    	<button type="button" class="btn btn-primary">
+                    <button type="button" class="btn btn-primary">
 		      		<a href="https://docs.google.com/forms/d/e/1FAIpQLSeFPQAybTSy5sIGczZ5tK-2ySrscvWfpq01U3dol4wFy3nIEw/viewform" target="_blank">Submeta um evento Próprio</a>
 		    	</button>
-          	</div>
-      </div>
+                </div>
+            </div>
     </section>
 
- 
+
     <footer style="background-color:#d67f7f;">
```

Comment:
Adiciona esse id ao `footer` para que quando a pessoa clique na aba `Contato` ela seja redirecionada para o footer:

```suggestion
    <footer id="contact" style="background-color:#d67f7f;">
```